setup.py

```diff
@@ -9,7 +10,15 @@ def list_files(directory):
         listing.append(pair)
     return listing
 
-VERSION = '2.2.5'
+def get_version()->str:
```

Comment:
why not importing the version?

Reply:
Implemented. It seems I have this irrational fear that _if the software isn't already installed you shouldn't rely on it_. 

By the way, the __init__.py files are automatically imported when the package name is invoked, so this works:

```
from ldm.invoke import __version__ as version
```

Reply:
The Reason why I recommended to only import the __init__ file was to not need importing the whole package, since you where only interested in the version number 😅

Reply:
There's no separate invoke.py package. __init__.py is all there is!

Reply:
Ahhhhh - I see - sorry, now I get the `from ldm.invoke import __version__ as version` statement (poor developers, always have to deal with those non-dev people like me xD )